git-cliff/src/args.rs

```diff
@@ -146,10 +146,13 @@ pub struct Opt {
 		value_name = "TAG",
 		allow_hyphen_values = true
 	)]
-	pub tag:          Option<String>,
+	pub tag:            Option<String>,
 	/// Bumps the version for unreleased changes.
 	#[arg(long, help_heading = Some("FLAGS"))]
-	pub bump:         bool,
+	pub bump:           bool,
+	/// Returns bumped version for unreleased changes.
+	#[arg(long, help_heading = Some("FLAGS"))]
+	pub bumped_version: bool,
```

Comment:
Not sure if I like the argument name. We can go with `--print-version` or something similar. What do you think?